packages/components/src/radio-control/index.js

```diff
@@ -53,6 +54,7 @@ export default function RadioControl( {
 							aria-describedby={
 								!! help ? `${ id }__help` : undefined
 							}
+							{...props}
```

Comment:
Linter complain about this line:

> [1] /home/runner/work/gutenberg/gutenberg/packages/components/src/radio-control/index.js
78
> [1]   57:9  error  Replace `...props` with `·...props·`  prettier/prettier


```suggestion
							{ ...props }
```